src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -105,6 +105,9 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // AppBar
                 (new ExternalThemeKeyList(typeof(AppBar), typeof(AppBarStyle))),
+
+                // NumberPicker
+                (new ExternalThemeKeyList(typeof(NumberPicker), typeof(NumberPickerStyle))),
```

Comment:
This is for external json theme injection.
Defining just `ExternalThemeKeyList` is meaningless. (AppBar should be fixed also)
Please add proper properties you want to set, for example,

```C#
(new ExternalThemeKeyList(typeof(NumberPicker), typeof(NumberPickerStyle)))
    .Add<Size2D>("/StartScrollOffset", (ViewStyle style, Size2D value) => ((NumberPickerStyle)style).StartScrollOffset = value)),
    .Add<Size>("/CenterLine/Size", (ViewStyle style, Size value) => ((NumberPickerStyle)style).CenterLine.Size = value))
```
This can enabled injection of json data, such as
```json
{
  "NumberPicker": {
    "StartScrollOffset" : "0, 12",
    "CenterLine": {
      "Size" : "160, 2", 
    }
  }
}
```

Reply:
Thank you for the guide.